pkg/router/httpTriggers.go

```diff
@@ -423,6 +423,7 @@ func (ts *HTTPTriggerSet) updateRouter(ctx context.Context) {
 				fn := *f.(*fv1.Function)
 				functionTimeout[fn.ObjectMeta.UID] = fn.Spec.FunctionTimeout
 				allfunctions = append(allfunctions, fn)
+				ts.logger.Info("allFunctions count %d", zap.Int("count", len(allfunctions)))
```

Comment:
All arguments after the message are key/value pairs, so this would end up logging:
```json
{ "message": "allFunctions count %d", "count": 123 }
```

I'd suggest dropping this log and just logging the trigger and function counts in the `updated router` log below.